langlib/lang.table/src/main/java/org/ballerinalang/langlib/table/RemoveAll.java

```diff
@@ -32,10 +33,9 @@ public class RemoveAll {
     public static void removeAll(BTable tbl) {
         try {
             tbl.clear();
-        } catch (io.ballerina.runtime.internal.util.exceptions.BLangFreezeException e) {
+        } catch (BError e) {
```

Comment:
We could simply remove the try catch block right? `tbl.clear()` will anyway give a runtime exception and it will be consistent with the errors that we get when we call langlib methods like.. put.. etc

Reply:
We'll have to handle all functions or neither. @lasinicl can you check the rest of the langlibs array, map, etc